src/app/components/BackButton/BackButton.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { ReactNode } from "react";
+import styles from "./BackButton.module.css";
+
+type BackButtonProps = {
+  children: ReactNode;
+};
+
+function BackButton({ children }: BackButtonProps): JSX.Element {
```

Comment:
Actually, this is not a `BackButton`, it is a `Button`.
You can use this component to create a `BackButton` (which might have an `onClick` and specific children`).